src/v0/destinations/ga4/networkHandler.js

```diff
@@ -30,9 +30,9 @@ const responseHandler = (destinationResponse, dest) => {
       const { description, validationCode, fieldPath } = response.validationMessages[0];
       throw new NetworkError(
         `Validation Server Response Handler:: Validation Error for ${dest} of field path :${fieldPath} | ${validationCode}-${description}`,
```

Comment:
@mihir-4116 I don't see any changes in GA4 test cases - so this scenario itself is missing, get it added by mocking destination response with failure message

Reply:
added ga4 proxy tests